### PR TITLE
Fix French Locales

### PR DESCRIPTION
**Pull request by netouss**
Add text coloration and translation.

### CODE REVIEW

File: locales/fr.lua [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-	['message_sent'] = 'message envoyé',
-	['cannot_add_self'] = 'vous ne pouvez pas vous ajouter vous-même',
-	['number_in_contacts'] = 'ce numéro est déja dans votre liste de contacts',
-	['contact_added'] = 'contact ajouté',
-	['contact_removed'] = 'contact supprimé',
-	['number_not_assigned'] = 'ce numéro n\'est pas attribué...',
-	['invalid_number'] = 'that\'s not an valid number!',
+	['gps_position'] = '~y~Position~s~ entrée ~g~~h~dans le GPS',
+	['message_sent'] = 'Message ~g~~h~envoyé',
+	['cannot_add_self'] = 'Vous ne pouvez pas ~r~~h~vous ajouter vous-même',
```
[the remark this answers is not in the document]
Comment (netouss) [reply locales/fr.lua]: Oh, ok, i didn't knew that sorry. Why starting lower case ? It's a rule ? Thanks in advance ;)

File: locales/fr.lua [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Locales['fr'] = {
 	['new_message'] = '~b~Nouveau message :~s~ %s',
-	['press_take_call'] = '%s - Appuyez sur ~INPUT_CONTEXT~ pour ~g~~h~prendre l\'appel',
```
Comment (majormarcin) [inline locales/fr.lua]: this can be started with a capital letter

File: locales/fr.lua [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+	['press_take_call'] = '%s - appuyez sur ~INPUT_CONTEXT~ pour ~g~~h~prendre l\'appel',
 	['taken_call'] = '~y~%s~s~ a pris l\'appel',
-	['gps_position'] = '~y~Position~s~ entrée ~g~~h~dans le GPS',
-	['message_sent'] = 'Message ~g~~h~envoyé',
-	['cannot_add_self'] = 'Vous ne pouvez pas ~r~~h~vous ajouter vous-même',
-	['number_in_contacts'] = 'Ce numéro est ~r~~h~déja dans votre liste de contacts',
-	['contact_added'] = 'Contact ~g~~h~ajouté',
-	['contact_removed'] = 'Contact ~r~~h~supprimé',
+	['gps_position'] = '~y~position~s~ entrée ~g~~h~dans le GPS',
+	['message_sent'] = 'message ~g~~h~envoyé',
```
Comment (majormarcin) [inline locales/fr.lua]: it must be started with a lowercase letter

File: locales/fr.lua [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-	['press_take_call'] = '%s - Appuyez sur ~INPUT_CONTEXT~ pour ~g~~h~prendre l\'appel',
+	['press_take_call'] = '%s - appuyez sur ~INPUT_CONTEXT~ pour ~g~~h~prendre l\'appel',
 	['taken_call'] = '~y~%s~s~ a pris l\'appel',
-	['gps_position'] = '~y~Position~s~ entrée ~g~~h~dans le GPS',
-	['message_sent'] = 'Message ~g~~h~envoyé',
-	['cannot_add_self'] = 'Vous ne pouvez pas ~r~~h~vous ajouter vous-même',
-	['number_in_contacts'] = 'Ce numéro est ~r~~h~déja dans votre liste de contacts',
-	['contact_added'] = 'Contact ~g~~h~ajouté',
-	['contact_removed'] = 'Contact ~r~~h~supprimé',
+	['gps_position'] = '~y~position~s~ entrée ~g~~h~dans le GPS',
```
Comment (majormarcin) [inline locales/fr.lua]: it can also start with a capital letter because it starts with ~ somethin ~
where something means 1 letter of color in English

**Pull request closed without merging**